applications/SwimmingDEMApplication/python_scripts/cellular_flow/product_of_sines_benchmark_analysis.py

```diff
@@ -1,6 +1,6 @@
 from KratosMultiphysics import Parameters
 import KratosMultiphysics.SwimmingDEMApplication as SDEM
-import ethier_benchmark_analysis
+import cellular_flow.ethier_benchmark_analysis as ethier_benchmark_analysis
```

Comment:
same

Reply:
Done in next push. However it's not in the tests, maybe it's deprecated.